platform/linux-generic/pktio/loop.c

```diff
@@ -191,7 +191,7 @@ static int loopback_recv(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 	}
 
 	pktio_entry->s.stats.in_errors += failed;
-	pktio_entry->s.stats.in_ucast_pkts += num_rx - failed;
```

Comment:
Don't *_ucast_pkts still need to be calculated?

Reply:
The previous calculation of unicast packets was a bit faulty. The destination MACs were never checked. I'd prefer leaving `in_ucast_pkts` and `out_ucast_pkts` unsupported on loop interface to avoid extra overhead.

Reply:
And pcap as well, right? Okay, sounds reasonable to me.

Reply:
Yep, same thing.